config/ingest.go

```diff
@@ -55,6 +55,8 @@ type Ingest struct {
 	// MinimumKeyLengt causes any multihash, that has a digest length less than
 	// this, to be ignored.
 	MinimumKeyLength int
+	// OverwriteMirrorOnResync overwrites the advertisement when resyncing.
+	OverwriteMirrorOnResync bool
```

Comment:
At some point we probably this configurable per `admin sync --resync` command as well?

Reply:
Agree, but that will slightly more complicated because of how the resync flag is passed through. I can do that in a separate PR. https://github.com/ipni/storetheindex/issues/2031

Reply:
Totally; separate all PRs ✊